cl/search/models.py

```diff
@@ -3050,6 +3060,66 @@ def sort_cites(c):
         return 8
 
 
+OPINION_TEXT_SOURCE_FIELDS = [
+    "html_with_citations",
+    "xml_harvard",
+    "html_columbia",
+    "html_lawbox",
+    "html_anon_2020",
+    "html",
+]
+
+
+class OpinionQuerySet(models.QuerySet):
```

Comment:
Should we start using this wherever we're getting the text of the decision? Because some opinions are so large, I bet we could really make large pages faster by doing this. Is that right? If so, can we open a bug to do so?

Reply:
Sure. Here is the issue to follow up this one: https://github.com/freelawproject/courtlistener/issues/5374